app/frontend/src/submissions/components/SubmissionForm/Location.vue

```diff
@@ -38,10 +38,23 @@ Licensed under the Apache License, Version 2.0 (the "License");
             id="wellStreetAddress"
             type="text"
             label="Street address"
+            @input="fetchAddressSuggestions"
+            v-on:focus="showList(true)"
+            v-on:blur="showList(false)"
             :errors="errors['street_address']"
             :loaded="fieldsLoaded['street_address']"
             :disabled="sameAsOwnerAddress"
           ></form-input>
+           <!-- Display the address suggestions -->
+        <div v-if="addressSuggestions.length > 0" class="address-suggestions list-group list-group-flush border" id="address-suggestions-list">
```

Comment:
you may need to change the name of ID `address-suggestions-list` to something more unique to its use case, its being declared in two separate files. 
Unless I am mistaken, both these pages render on the same page in the application, and the reuse of the same id can pose problems in the DOM